diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -373,11 +379,12 @@ CallbackReturn DiffDriveController::on_configure(const rclcpp_lifecycle::State &
       std::make_shared<realtime_tools::RealtimePublisher<Twist>>(limited_velocity_publisher_);
   }
 
-  received_velocity_msg_ptr_ = std::make_shared<Twist>();
+  const Twist empty_twist;
+  received_velocity_msg_ptr_.set(std::make_shared<Twist>(empty_twist));
```

Comment:
does a `shared_ptr` and the `emplace` below play well together in this case?

Reply:
The `previous_commands_` stores Twist objects, not poitners. I think it should be fine.

Reply:
Ah! Right. `make_shared` creates a copy that it manages